docs/DeveloperGuide.md

```diff
@@ -788,37 +802,70 @@ testers are expected to do more *exploratory* testing.
 
 1. Deliverables, meetings and contacts are saved automatically to ./data/.
 
-   On normal usage, 3 JSON files are created:
-      * `contactbook.json`
-      * `meetingbook.json`
-      * `deliverablebook.json`
-
+   On normal usage, 3 JSON files are created / saved - `contactbook.json`, `meetingbook.json` and `deliverablebook.json`.
    All 3 files contain information stored by the user from their respective modes.
 
    On first starting the program, a file is only created if the user inputs a command specific to that mode.
 
    1. Prerequisites: Very first time using the app.
 
-   1. Test case: Starting and close the app immediately.<br>
-      Expected: No JSON files created.
+   1. Test case: Start and close the app immediately.<br>
+      Expected: The 3 JSON files are not created.
 
-   1. Test case: Start the app. Switch to deliverable mode. Add a deliverable. Close the app.
-      Expected: Only `deliverablebook.json` created.
+   1. Test case: Start the app. Switch to deliverable mode. Add a deliverable. Close the app.<br>
+      Expected: Of the 3 JSON files, only `deliverablebook.json` created.
```

Comment:
Can edit to "only `deliverablebook.json` is created."

Reply:
Concerned that it might be misinterpreted as preferences.json and a bunch of other .json files will be created